portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,14 @@
+package com.google.sps.data;
+
+public final class Comment {
+
+  private String commentText = "Placeholder Comment";
```

Comment:
I'd leave these uninitialized, since you don't have a default constructor anyway

Reply:
Done.